.travis.yml

```diff
@@ -42,7 +55,7 @@ before_script:
 
 script:
   - ./vendor/bin/phpcs -s --standard=phpcs.ruleset.xml
-  - phpunit
+  - bash bin/test.sh
   - yarn run build
   - npm test
 
```

Comment:
I noticed in the travis build `deploy->skip_cleanup` is now deprecated. It suggests using `cleanup` instead.
https://docs.travis-ci.com/user/deployment-v2#cleaning-up-the-git-working-directory

Reply:
I saw that too and was going to update it, but it mentions that it's for dpl v2 and I honestly wasn't sure what that was and figured it would be hard to tell if it broke anything until we ran the `deploy` flow by merging the PR.

It will probably be safer in the long-run to update it now though.